packages/developer-portal/src/components/pages/webhooks/webhooks-manage-form.tsx

```diff
@@ -0,0 +1,310 @@
+            <Label className={elMl3}>Subscription topics</Label>
+            <InputGroup
+              className={searchMinWidth}
+              onChange={handleSearchTopics(topics, getValues, setFilteredTopics)}
+              icon="searchSystem"
+              placeholder="Search"
+            />
+          </FlexContainer>
+          <MultiSelectInput
+            className={elMb7}
+            id={`topic-edit-ids-${id}`}
+            hasGreyChips
+            defaultValues={[...new Set(topicIds)]}
+            options={topicOptions}
+            {...register('topicIds')}
+          />
+          <Label className={elMl3}>Subscription customers</Label>
+          <MultiSelectInput
+            className={elMb7}
+            id={`customer-edit-ids-${id}`}
+            hasGreyChips
+            defaultValues={[...new Set(customerIds)]}
+            options={customerOptions}
+            {...register('customerIds')}
+          />
+        </ColSplit>
+        <ColSplit>
+          <Grid>
+            <ColSplit>
+              <Label>Status</Label>
+              <Toggle
+                className={cx(elMt1, elMb11)}
+                id={`status-edit-toggle-${id}`}
+                hasGreyBg
+                isFullWidth
+                {...register('active')}
+              >
+                <span className={elToggleItem}>Active</span>
+                <span className={elToggleItem}>Inactive</span>
+              </Toggle>
+            </ColSplit>
+          </Grid>
+          <InputGroup
+            className={elMb11}
+            type="checkbox"
+            label="Ignore where only the etag has been modified"
+            inputAddOnText="Ignore"
+            {...register('ignoreEtagOnlyChanges')}
+          />
+          <ButtonGroup className={elMlAuto}>
+            <Button
+              intent="danger"
+              type="button"
+              disabled={webhookCreateEditState === WebhookCreateEditState.LOADING}
+              onClick={openModal}
+            >
+              Delete
+            </Button>
+            <Button
+              intent="secondary"
+              type="button"
+              disabled={webhookCreateEditState === WebhookCreateEditState.LOADING}
+              onClick={handleCollapseRow(setIndexExpandedRow, setExpandableContentSize)}
+            >
+              Cancel
+            </Button>
+            <Button
+              intent="primary"
+              chevronRight
+              type="submit"
+              disabled={webhookCreateEditState === WebhookCreateEditState.LOADING}
+            >
+              Update
+            </Button>
+          </ButtonGroup>
+          <DeleteConfirmModal title="Delete Webhook">
+            <BodyText hasGreyText>Are you sure you want to delete this webhook?</BodyText>
+            <BodyText>{url}</BodyText>
+            <BodyText hasGreyText>
+              By clicking ‘Delete’ it will remove the webhook and all its data Please click ‘Delete’ to continue with
```

Comment:
not sure on this one?
```suggestion
              By clicking ‘Delete’ the system will remove the webhook and it's stored configurations Please click ‘Delete’ to continue with
```